src/main/java/codeu/model/data/Conversation.java

```diff
@@ -26,6 +26,7 @@
   public final UUID owner;
   public final Instant creation;
   public final String title;
+  private String type;
```

Comment:
This is the sort of thing where a boolean makes things safer. You could easily mistype the value of the string field in other places in the code. How about making this `private Boolean isPrivate;`?

Reply:
I agree that a boolean would be safer

Reply:
Yes, you're right. will change it.